.eslintrc.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     'new-cap': [
       'error',
       {
-        capIsNewExceptionPattern: '(\\.|^)EXPERIMENTAL_.+',
+        capIsNewExceptionPattern:
```

Comment:
This lets us have capitalized components (e.g., as exposed on `hits` and `infiniteHits`).